PathsActivity.py

```diff
@@ -304,12 +304,12 @@ def _list_tubes_reply_cb(self, tubes):
 
     def _list_tubes_error_cb(self, e):
         """ Log errors. """
-        print('Error: ListTubes() failed: %s', e)
+        print(('Error: ListTubes() failed: %s', e))
```

Comment:
I'm guessing you used `2to3` and didn't make any changes after that, this line would print both `%s` and the value of `e`.
```python
Error: ListTubes() failed: %s e
```
There's no need for the double parentheses except you were wrapping the placeholder value which you're not here, 
the line should be
```python
print('Error: ListTubes() failed: %s' %(e))
```
or you can use f strings and it'll look like
```python
print(f'Error: ListTubes() failed: {e}')

Reply:
I seemed to have skipped that.... resolved in commit `aa4103e`